doc/release-notes/NEXT.md

```diff
@@ -0,0 +1 @@
+* When cloning a whole tfs repository (``$/``) without specifying a local repository name, infer the name from the host part of the server url (#1202)
```

Comment:
Maybe we should use the TFS terminology here. I believe the correct terminology is "TFS Project Collection":

> * When cloning a whole TFS Project Collection (``$\``) without specifying a local repository name,...